File: ios/BFExplorer/Plaoc/IPCCore/sys/mwebview/ViewItem.swift
//
//  ViewItem.swift
//  BFExplorer
//
//  Created by ui03 on 2023/3/8.
//

import UIKit

class ViewItem {

    var webviewId: String
    var webView: DWebView
    var hidden: Bool = false
    
    init(webviewId: String, dWebView: DWebView, hidden: Bool = false) {
        self.webviewId = webviewId
        self.webView = dWebView
        self.hidden = hidden
    }
}

File: android/DWebView/src/main/java/org/dweb_browser/dwebview/base/ViewItem.kt
package org.dweb_browser.dwebview.base

import com.google.accompanist.web.WebViewNavigator
import com.google.accompanist.web.WebViewState
import kotlinx.coroutines.CoroutineScope
import org.dweb_browser.dwebview.DWebView

data class ViewItem(
  val webviewId: String,
  val webView: DWebView,
  val state: WebViewState,
  val navigator: WebViewNavigator,
  val coroutineScope: CoroutineScope,
  var hidden: Boolean = false
)